e2e_samples/dataset_versioning/datafactory/README.md

```diff
@@ -0,0 +1,22 @@
+# Overview
+
+This folder is for Data pipeline which copy data from SQL DB to Azure Storage (Delta Lake). We are using Azure Data Factory to implement pipeline.
+
+## How to deploy it
+
+1. Provision Azure Resources by IaC (Terraform)
```

Comment:
I think these first 2 items are not steps to deploy. Maybe they are just prerequisites since I did them before reaching this page.

Reply:
Have updated based on this feedback. See detail in 
https://github.com/Azure-Samples/modern-data-warehouse-dataops/pull/305